### ISSUE TITLE
Save as seems to rip out styles from the worksheet after modifying values

### ISSUE THREAD

**Issue by bnandaku** [qax-os/excelize#1201]
When I add values to cells for worksheet that has some styles int. Even if I don't modify the cells that have styling. After saving the file. I lose all my styling. 

**Steps to reproduce the issue:**
1.  Open a spreadsheet with styles in some cells (Text is bold)
2. Add some values to cells that are not the cells with styling in them
3. Save file
4. Open file in Excel. Notice the bold styled cells have lost their styling

**Describe the results you received:**
Bold styled cells lose their styling

**Describe the results you expected:**
Bold styled cells should not lose the styling
**Output of `go version`:**

```text
go version go1.17.2 darwin/amd64
```

**Excelize version or commit ID:**

```text
github.com/xuri/excelize/v2 v2.5.0
```

**Environment details (OS, Microsoft Excel™ version, physical, etc.):**

Mac OS,Intel, Excel version 16.53

**Comment by xuri** [comment]
Thanks for your feedback, this issue is similar with #1119 and #1130, and has been fixed with [`236ee61`](../commit/236ee61d201e45b1fe33a58b290adb7ee32d3488), please upgrade to the master branch code, and this patch will be released in the next version. The next release is planned for Monday, April 11th, 2022.

**Comment by xuri** [comment]
The v2.6.0 has been released. I will close this issue. If you have any questions, please let me know.
